test/caller-test.jl

```diff
@@ -0,0 +1,46 @@
+function test_caller()
+    print("caller...")
+
+    R, (x, y) = PolynomialRing(Singular.ZZ, ["x", "y"])
+    @test Singular.SETS.isEqualInt(R, Singular.ZZ(1), Singular.ZZ(1)) == 1
+
+    R, (x, y) = PolynomialRing(Singular.QQ, ["x", "y"])
+
+    # Input tests
+    @test Singular.SETS.isEqualInt(R, 1, 1)                           == 1
+    @test Singular.SETS.isEqualInt(R, x, x)                           == 1
+    @test Singular.SETS.isEqualInt(R, "aa", "aa")                     == 1
+    @test Singular.SETS.isEqualInt(R, Singular.QQ(1), Singular.QQ(1)) == 1
+    @test Singular.SETS.isEqualInt(R, [1,2,3], [1,2,3])               == 1
+    @test Singular.SETS.isEqualInt(R, [1 2; 3 4], [1 2; 3 4])         == 1
```

Comment:
Perhaps rewriting these tests with the order of the `==` arguments switched would be easier to maintain? I.e.
```suggestion
    @test 1 == Singular.SETS.isEqualInt(R, [1 2; 3 4], [1 2; 3 4])
```
But of course it's fine as it is, too